content/building/environment-variables.md

```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
 4. Click **Save** to save the settings.
 
 ![](../uploads/env_vars.PNG)
+
+## Accessing environment variables from your application
+
+The next examples show how to place your Google Maps API key into an Android or iOS application from an environment variable. Doing it allows you not to store your secret key in the source code.
+
+1. Add your key as an environment variable **MAPS_API_KEY**
+
+### Android
+
+2. Read your key from an environment variable to **build.gradle**
+
+        defaultConfig {
+            // Other values set here
+            resValue "string", "maps_api_key", "$System.env.MAPS_API_KEY"
+        }
+
+3. Read your key from a **build.gradle** value to **AndroidManifest.xml**
```

Comment:
Read your key > Read the key from the **build.gradle** value...

Reply:
Use inline code (backticks) format for file names?